tests/lac/utilities_04.output

```diff
@@ -0,0 +1,7 @@
+
+DEAL::0.00000
+DEAL::0.00000
+DEAL::0.00000
+DEAL::0.00000
+DEAL::0.00000
+DEAL::0.00000
```

Comment:
I have not tried to figure out the Givens rotations for the particular values you give but is it intended that all results are zero?

Reply:
yes, it checks that `Rotation * [x y]^T = [r 0]^T` and then calculates residual of this equation.

Reply:
Ah, now I see, then this makes perfect sense. I admit I didn't read carefully enough. Maybe help future readers of the test case by adding the word `residual` to the output? (But that may be just my taste, I'm equally fine if you leave things as they are.)

Reply:
done.